source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
 ---
-title: How accounts work
-last_reviewed_on: 2021-01-08
+title: Setting up multiple services
+last_reviewed_on: 2021-02-03
 review_in: 6 months
 weight: 40
 ---
 
-# How accounts work
+# Setting up multiple services
+  
+If you have more than one online service, you can set up your GOV.UK Pay account and payment service provider (PSP) accounts so it's easier to manage your payments and services.
 
-GOV.UK Pay does not limit how many services you can integrate with it and how many accounts you can have with us.
-GOV.UK Pay test and live accounts are free to create and maintain but your payment service provider may charge you for the underlying accounts.
+## Decide where to send users
 
-## Account structure examples
+You can send your users to make payments with either:
 
-If you do integrate multiple services, you can treat them separately or as a
-combined single service within GOV.UK Pay and your payment service provider (PSP).
-This gives 3 possible account configurations.
+- a different GOV.UK Pay service for each of your services
+- a combined GOV.UK Pay service
 
-As an example, take 2 services called ‘parking permits’ and ‘parking fines’:
+### Send your users to different GOV.UK Pay services
 
-### Type 1 (GOV.UK Pay separate, PSP separate)
+Send your users to a different GOV.UK Pay service for each of your services if you need to do any of the following:
 
-<%= image_tag "/images/accountstructure_1.svg", { :alt => '' } %>
+- give each service its own name on the GOV.UK Pay payment pages
+- report on each service's payments separately
+- send each service's payments to different bank accounts
+- give your staff members access to each service separately in the GOV.UK Pay admin tool
 
```

Comment:
maybe we also need something about being able to manage settings individually on different accounts? e.g. what card types are accepted, the custom paragraph in the confirmation email. Do we have a link to those in the docs or would the Settings page in the admin tool be the best place to see all of those?

Reply:
oh and we should add that if you are using Worldpay and need to take MOTO payments, you need a separate service for that (as you need a separate merchant code for ECOMM vs. MOTO https://docs.payments.service.gov.uk/moto_payments/#take-a-payment-over-the-phone-moto-payments

Reply:
On the first point: good point. I've added this. I don't think we need to list the settings - they're clear from the admin tool.

On the second point: I'm not sure we could get into MOTO here. It's not going to be a criteria for many users that affects their decisions, and we give specific instructions for creating a separate account "if you do not have one already" on the MOTO page itself.